src/docs.js

```diff
@@ -4,44 +4,60 @@ let assert = require('assert');
 let fs = require('mz/fs');
 let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
+let zlib = require('zlib');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
+    metadata: null,
     schemas: [],
     docsFolder: null,
+    references: [],
   });
   assert(options.schemas, 'options.schemas must be given');
   assert(options.schemas instanceof Array, 'options.schemas must be an array');
 
-  let schemas = options.schemas;
   let tarball = tar.pack();
+  let metadata = options.metadata;
```

Comment:
I think we should set the `version: 1` bits internally to the library rather than needing them to be passed in. I think I like that you've added the ability to set other fields of the metadata, but at least we need to set the version automatically.